src/components/resource-picker/storage-location-screens.js

```diff
@@ -94,17 +87,11 @@ function StorageLocationScreens({ storageLocation, initialUrl, onSelect, onCance
 
   const handleDirectoryClick = newFile => {
     setIsLoading(true);
-    setCurrentDirectoryPath(newFile.path);
+    setContentFetchingProps({ searchTerm: '', currentDirectoryPath: newFile.path });
   };
 
   const handleFileDoubleClick = newFile => {
-    if (newFile.type === CDN_OBJECT_TYPE.file) {
-      onSelect(newFile.portableUrl);
-    }
-    if (newFile.type === CDN_OBJECT_TYPE.directory) {
-      setIsLoading(true);
-      setCurrentDirectoryPath(newFile.path);
-    }
```

Comment:
This was some leftover code that was actually never reached/executed.

Reply:
:+1: